src/app/listener.ts

```diff
@@ -42,6 +45,29 @@ export class Listener {
 
       this.container.loggerService.info(`Loaded ${this.container.jobService.size()} jobs...`);
 
+      const commands = Object.entries(this.container.pluginService.plugins)
+        .filter((entry) => {
+          const [, plugin] = entry;
+          return isSlashCommand(plugin);
+        })
+        .map((entry) => {
+          const [name, plugin] = entry;
+          const options = isSlashCommand(plugin)
+            ? (plugin as unknown as ISlashPlugin).parameters
```

Comment:
since you filter, do you need to do this?

Reply:
Don't think TS updates types of an array after using a filter

Reply:
cleaner to do a map after the filter then, if filter_map doesnt exist in ts